src/test/resources/draft2020-12/issue495.json

```diff
@@ -1,61 +1,105 @@
 [
     {
         "description": "issue495 using ECMA-262",
-        "regex": "ecma-262",
```

Comment:
I moved this fields, because `regex` is defined in the `TestSpec` class, not inside `TestCase`. Without this the tests were skipped...